docs/2.5. APIs - Query Parameters.md

```diff
@@ -317,160 +346,180 @@ In this case, assume that the most recently created or updated Node held in the
-* Returns Nodes with versions greater than or equal to v1.2, and less than v2.0. These Nodes must have all non-v1.1 keys stripped by the Query API before they are returned.
+
+- Returns all Nodes held in the registry which match the v1.1 schema.
+- Returns Nodes with versions greater than or equal to v1.2, and less than v2.0. These Nodes MUST have all non-v1.1 keys stripped by the Query API before they are returned.
 
 **Example 2: Downgrade From v1.1 to v1.0**
 
 ***Request***
-```
+
+```http
 GET /x-nmos/query/v1.1/nodes?query.downgrade=v1.0
 ```
 
 ***Response***
-* Returns all Nodes held in the registry which match the v1.1 schema.
-* Returns Nodes with versions greater than or equal to v1.2, and less than v2.0. These Nodes must have all non-v1.1 keys stripped by the Query API before they are returned.
-* Returns all Nodes which are registered in v1.0 format.
+
+- Returns all Nodes held in the registry which match the v1.1 schema.
+- Returns Nodes with versions greater than or equal to v1.2, and less than v2.0. These Nodes MUST have all non-v1.1 keys stripped by the Query API before they are returned.
+- Returns all Nodes which are registered in v1.0 format.
 
 **Example 3: Downgrade From v1.3 to v1.1**
 
 ***Request***
-```
+
+```http
 GET /x-nmos/query/v1.3/flows?query.downgrade=v1.1
 ```
 
 ***Response***
-* Returns all Flows held in the registry which match the v1.3 schema.
-* Returns Flows with versions greater than or equal to v1.4, and less than v2.0. These Flows must have all non-v1.3 keys stripped by the Query API before they are returned.
-* Returns all Flows which are registered in v1.2 or v1.1 format.
+
+- Returns all Flows held in the registry which match the v1.3 schema.
+- Returns Flows with versions greater than or equal to v1.4, and less than v2.0. These Flows MUST have all non-v1.3 keys stripped by the Query API before they are returned.
+- Returns all Flows which are registered in v1.2 or v1.1 format.
 
 ### Invalid Examples
 
 **Invalid Example 1: Downgrade Between Major API Versions**
 
 ***Request***
-```
+
+```http
 GET /x-nmos/query/v3.0/flows?query.downgrade=v1.0
 ```
 
 ***Response***
-* Returns an HTTP 400 error code as downgrade queries may not be performed between major API versions.
+
+- Returns an HTTP 400 error code as downgrade queries MUST NOT be performed between major API versions.
 
 ## Basic Queries
 
 Query APIs SHOULD support basic queries against their API resources. A 501 HTTP status code MUST be returned where a basic query is attempted against a Query API which does not implement it.
 
-Basic queries make use of standard HTTP GET query parameters in the form '?key1=value1&key2=value2'. Keys may match any attribute which the API schemas indicate could be returned by a given resource.
+Basic queries make use of standard HTTP GET query parameters in the form '?key1=value1&key2=value2'. Keys MAY match any attribute which the API schemas indicate could be returned by a given resource.
```

Comment:
This doesn't read quite right. This could be "MUST" (or "SHOULD" given the following sentence)? Or not use conformance language?

Reply:
As the first sentence in this para states a fact we can do the same for the second sentence.